src/protos/io/defang/v1/fabric.proto

```diff
@@ -117,6 +117,24 @@ message UploadURLRequest { string digest = 1; }
 
 message UploadURLResponse { string url = 1; }
 
+
+enum ServiceState {
+  UNKNOWN = 0;
+
+  // Build states (initial state for build)
+  BUILD_PENDING = 1; // BUILD_QUEUED -> BUILD_ACTIVATING
+  BUILDING = 2; // BUILD_RUNNING
+  CREATED = 3; // BUILD_DEACTIVATING (done = true)
+ 
+  // Update states (initial state for existing image)
+  UPDATE_PENDING = 4; //  UPDATE_QUEUE
+
+  // Deployment states
+  DEPLOYING = 5;
```

Comment:
especially these: PENDING, COMPLETED, FAILED are pretty clear, why change it up?